GitUI/CommandsDialogs/SettingsDialog/FormAvailableEncodings.cs

```diff
@@ -59,9 +59,22 @@ private void ToLeft_Click(object sender, EventArgs e)
         {
             if (ListAvailableEncodings.SelectedItem is not null)
             {
-                var index = ListAvailableEncodings.SelectedIndex;
+                Encoding selected = (Encoding)ListAvailableEncodings.SelectedItem;
+                foreach (Encoding included in ListIncludedEncodings.Items)
+                {
+                    if (included.WebName == selected.WebName)
+                    {
+                        MessageBox.Show(this,
```

Comment:
Just realised the icon is missing, please add a warning icon. And a heading.

Reply:
This check was too late anyway. I have reverted it. Instead, conflicting encodings are filtered out `LoadEncoding`.